tests/views/playingScreen-test.js

```diff
@@ -562,7 +587,12 @@ describe('PlayingScreen', function () {
       cueText: "cue text"
     };
 
-    var DOM = TestUtils.renderIntoDocument(<PlayingScreen  controller = {mockController} closedCaptionOptions={closedCaptionOptions} />);
+    var DOM = TestUtils.renderIntoDocument(
+      <PlayingScreen
+        controller = {mockController}
+        closedCaptionOptions={closedCaptionOptions}
+        handleVrPlayerMouseUp={handleVrPlayerMouseUp}
+      />);
     var unmuteIcons = TestUtils.scryRenderedComponentsWithType(DOM, UnmuteIcon);
     expect(unmuteIcons.length).toBe(0);
```

Comment:
Can we add unit tests for fixes and changes introduced in this PR (e.g. `START_LOADING_SCREEN`, new document event listeners, etc.)?

Reply:
\+

Reply:
+1 to adding new unit tests